### PR TITLE
add fairseq

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @ndmaxar agree to be maintainers?

File: recipes/fairseq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  host:
+    - python
+    - pip
+    - tqdm
+    - sacrebleu
+    - pytorch
+    - regex
+    - pycparser
+    - cython
+    - setuptools
```
Comment (scopatz) [inline recipes/fairseq/meta.yaml:37]: Do you need all of these in host?

File: recipes/fairseq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+  run:
+    - python
+    - cffi
+    - numpy
```
Comment (scopatz) [inline recipes/fairseq/meta.yaml:42]: Is numpy only used at runtime?

Comment (rluria14) [reply recipes/fairseq/meta.yaml:40]: added numpy to host

File: recipes/fairseq/meta.yaml (hunk 1 above)
Comment (rluria14) [reply recipes/fairseq/meta.yaml:34]: removed some unnecessary deps

**Comment by rluria14** [comment]
@conda-forge/staged-recipes this is ready for review

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**